test/cast_test.exs

```diff
@@ -130,6 +130,45 @@ defmodule OpenApiSpec.CastTest do
       assert error.path == [:age]
     end
 
+    test "additional properties" do
```

Comment:
It's probably better to move this test to `test/cast/object_test.exs`.

Reply:
Thanks, didn't see it!
I'll move those two tests there, since it's not exactly what the other tests are doing: this is about resolving references (that is the main point of the issue), while the already present tests use schemas defined in-place, so I think at least the first is useful (the nested one may be overkill since normal schema resolution is already well tested?)